### PR TITLE
chore(build): remove jcenter since it no longer exists

### PR DESCRIPTION
**Pull request by dbyron-sf**
see https://status.bintray.com/

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -23,15 +23,18 @@ plugins {
 
 allprojects {
   apply plugin: "io.spinnaker.project"
+
+  task listrepos {
+    doLast {
+      println "Repositories:"
+      project.repositories.each { println "Name: " + it.name + "; url: " + it.url }
+    }
+  }
```
Comment (link108) [inline build.gradle:32]: did you want this to be included in the PR?

**Review by link108 — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (dbyron-sf) [reply build.gradle:32]: Whoops...so much for `git add -p` when spotless is around.

Comment (dbyron-sf) [reply build.gradle]: Removed: https://github.com/spinnaker/kork/pull/871/commits/4da7dd983b54db1ea9b1a355b1ee034486d4f8ab

**Comment by link108** [comment]
@mergifyio backport release-1.25.x release-1.26.x

**Pull request merged**